### PR TITLE
Completed A/D2

### PR DESCRIPTION
**Pull request by itz-ankit**
Sorry for the dot comments :')

### CODE REVIEW

**Review by ArchismwanChatterjee — CHANGES_REQUESTED** [review]
The question is saying to accept a number and check. There will be no output for numbers greater than 9. So please prefer using a general solution.

File: A/D2/even_odd.c
Comment (Gadzrux) [inline A/D2/even_odd.c:1]: I think the question asks us to check if a number is even or not and the number doesn't necessarily have to be within 1-10

File: A/D2/greatest.c
Comment (Gadzrux) [inline A/D2/greatest.c:1]: the numbers don't have to be distinct try using >= operators instead of >

**Review by Gadzrux — CHANGES_REQUESTED** [review]
Thanks for contributing please look into the changes that are suggested :)

File: A/D2/even_odd.c
Comment (ArchismwanChatterjee) [reply A/D2/even_odd.c:1]: Yes

File: A/D2/greatest.c
Comment (ArchismwanChatterjee) [reply A/D2/greatest.c:1]: Yes

**Comment by ArchismwanChatterjee** [comment]
It would be better if you format your code too

File: A/D2/even_odd.c
Comment (itz-ankit) [reply A/D2/even_odd.c:1]: sorry sorry , my bad :')

Comment (itz-ankit) [reply A/D2/even_odd.c:1]: I changed it
you can check it and lemme know if there's anything more I need to change

**Comment by Gadzrux** [comment]
All good thanks for contributing , have a great day

**Pull request merged**